website/public/policy-templates/windows/firezone_en-US.adml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<policyDefinitionResources revision="1.0" schemaVersion="1.0">
+  <displayName />
+  <description />
+  <resources>
+    <stringTable>
+      <string id="firezone">Firezone</string>
+      <string id="authURL">Authentication URL</string>
+      <string id="apiURL">WebSocket API URL</string>
+      <string id="logFilter">RUST_LOG filter string</string>
+      <string id="accountSlug">Account Slug</string>
+      <string id="internetResourceEnabled">Enforce full-tunnel</string>
+      <string id="hideAdminPortalMenuItem">Hide admin portal link</string>
+      <string id="connectOnStart">Connect on start</string>
+      <string id="startOnLogin">Start on login</string>
+      <string id="disableUpdateCheck">Disable update check</string>
+      <string id="supportURL">Support URL</string>
+
+      <string id="SUPPORTED_FZ_GUI_1_4_15">
+        Firezone GUI Client 1.4.15 or later
+      </string>
+
+      <string id="authURL_explain">
+        The base URL to open when users sign in. The accountSlug will be appended to this. In most cases you shouldn't change this. By default, the Client will use `https://app.firezone.dev`.
```

Comment:
I'm not sure how backticks are treated in XML, probably better just to use plain values (or wrap with `&quot;`?)
```suggestion
        The base URL to open when users sign in. The accountSlug will be appended to this. In most cases you shouldn't change this. By default, the Client will use https://app.firezone.dev.
```